sockets/1.2/modules/ROOT/pages/index.adoc

```diff
@@ -1,92 +1,54 @@
-	</sockets:listener-config>
-----
-
-
-== Supported Operations
-
-Socket Connector provides two basic operations:
-
-    * Send
-    * Send and Receive
-
-== Supported Protocols
-
-Sockets Connector supports these TCP protocols:
-
-* Direct Protocol +
-  The socket reads until no more bytes are (immediately) available. On slow networks, `EOFProtocol` and `LengthProtocol` might be more reliable.
-* EOF Protocol +
-  Reading is terminated by the stream being closed by the client.
-* Length Protocol +
-  This protocol is defined by sending or reading an integer (the packet length) and then the data to be transferred.
-* Custom Class Loading Length Protocol +
-  A length protocol that uses a specific class loader to load objects from streams.
-* Safe Protocol +
-  A test protocol that precedes every message with a cookie, and should not be used in production environments.
-* Streaming Protocol +
-  Allows the socket's Send operation to return a message with the original `InputStream` as payload.
-* XML Message Protocol +
-  The `XmlMessageProtocol` is a protocol used to read streaming XML documents. The only requirement is that each document includes an XML declaration at the beginning of the document of the form "<?xml...". Make sure that the XML documents being streamed begin with an XML declaration when using this protocol. +
-  Data is read until a new document is found or until there is no more currently available data. For slower networks, `XmlMessageEofProtocol` might be more reliable. +
-  Also, because the default character encoding for the platform is used to decode the message bytes when looking for the XML declaration, some caution with message character encodings is warranted
-* XML Message EOF Protocol +
-  Extends `XmlMessageProtocol` to continue reading until either a new message or EOF is found.
-* Custom Protocol +
-  Define your own custom protocol by writing a class that extends `TcpProtocol`.
-
-These application level protocols are implemented on top of TCP. For these protocols to work properly, implement these in both client and server ends.
-
-By default, a socket implements Safe Protocol, but you can specify a different protocol in the `config` element. The following example, demonstrates how to configure the socket to use Direct Protocol:
-
-[source,xml,linenums]
-----
-	<sockets:listener-config name="Sockets_Listener_config" doc:name="Sockets Listener config" >
-		<sockets:tcp-listener-connection host="localhost" port="8082" >
-			<sockets:protocol>
-				<sockets:direct-protocol />
-			</sockets:protocol>
-		</sockets:tcp-listener-connection>
-	</sockets:listener-config>
-----
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements in a Mule flow
+* Global elements
+* How to create a Mule app using Anypoint Studio
+
+Before creating an app, you must have access to Anypoint Platform and Anypoint Studio.
+
+== Common Use Cases For the Connector
```

Comment:
Since there's only one use case listed, perhaps rename this section to "Overview" or something similar?